image/memory.go

```diff
@@ -35,6 +35,15 @@ func (i *memoryImage) Reference() types.ImageReference {
 func (i *memoryImage) Close() {
 }
 
+// Size returns the size of the image as stored, if known, or -1 if not.
+func (i *memoryImage) Size() (int64, error) {
+	s, err := i.serialize()
+	if err != nil {
+		return -1, err
+	}
+	return int64(len(s)), nil
```

Comment:
This is only the length of the manifest, which does not really make sense. Considering that `memoryImage` is only returned by `types.Image.UpdatedImage`, it should say something semantically consistent with `sourcedImage.Size`.